sdk/cognitivelanguage/Azure.AI.Language.Conversations/src/autorest.md

```diff
@@ -8,15 +8,13 @@ title: Conversations
 license-header: MICROSOFT_MIT_NO_VERSION
 
 batch:
-- input-file: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/6e137f0849ff79637544c773ad6da9e7bff3faf1/specification/cognitiveservices/data-plane/Language/preview/2022-05-15-preview/analyzeconversations.json
+- input-file: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/e7f37e4e43b1d12fd1988fda3ed39624c4b23303/specification/cognitiveservices/data-plane/Language/preview/2022-05-15-preview/analyzeconversations.json
   clear-output-folder: true
-  model-namespace: false
-  data-plane: true
 
-# TODO: Uncomment when we ship authoring support and remove ./ConversationsClientOptions.cs.
-# - input-file: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/33138867cd88a4a8689feb591a98dda26d96a63e/specification/cognitiveservices/data-plane/Language/preview/2021-07-15-preview/analyzeconversations-authoring.json
-#   add-credentials: true
-#   data-plane: true
+- input-file: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/e7f37e4e43b1d12fd1988fda3ed39624c4b23303/specification/cognitiveservices/data-plane/Language/preview/2022-05-15-preview/analyzeconversations-authoring.json
+
+data-plane: true
+model-namespace: false
 
 modelerfour:
   lenient-model-deduplication: true
```

Comment:
Were we able to articulate why we need this flag?

Reply:
Errors abound /cc @AlexanderSher (he also asked in that email):

```
  error   | PreCheck/DuplicateSchema | Duplicate Schema named 'JobState' (5 differences):
    - properties: {"displayName":{"$ref":"#/components/schemas/schemas:228"},"createdDateTime":{"$ref":"#/components/schemas/schemas:229"},"expirationDateTime":{"$ref":"#/components/schemas/schemas:230"},"jobId":{"$ref":"#/components/schemas/schemas:231"},"lastUpdatedDateTime":{"$ref":"#/components/ 
  schemas/schemas:232"},"status":{"x-ms-enum":{"modelAsString":true,"name":"JobState"},"$ref":"#/components/schemas/JobState"},"errors":{"$ref":"#/components/schemas/schemas:234"},"nextLink":{"$ref":"#/components/schemas/schemas:235"}} => <none>
    - required: ["jobId","lastUpdatedDateTime","createdDateTime","status"] => <none>
    - type: "object" => "string"
    - enum: <none> => ["notStarted","running","succeeded","partiallyCompleted","failed","cancelled","cancelling"]
    - x-ms-enum: <none> => {"modelAsString":true,"name":"JobState"}; This error can be *temporarily* avoided by using the 'modelerfour.lenient-model-deduplication' setting.  NOTE: This setting will be removed in a future version of @autorest/modelerfour; schemas should be updated to fix this issue s 
  ooner than that.
  warning | PreCheck/PropertyRedeclarationWarning | Schema 'CurrencyResolution' has a property 'value' that is already declared the parent schema 'QuantityResolution' but isn't significantly different. The property has been removed from CurrencyResolution
EXEC : fatal   | error : 2 errors occured -- cannot continue. [C:\src\azure-sdk-for-net\sdk\cognitivelanguage\Azure.AI.Language.Conversations\src\Azure.AI.Language.Conversations.csproj]
```